src/structs.jl

```diff
@@ -234,13 +279,14 @@ struct Model
     compartments::Dict{String,Compartment}
     species::Dict{String,Species}
     initial_assignments::Dict{String,Math}
+    rules::Vector{Rule}
```

Comment:
This field is a bit of an outlier here: it's a vector, all others are either dictionaries or `Maybe{x}`.

Reply:
The rules do not have identifiers right? If not it's just a vector.

Reply:
:sob: tho.

Reply:
Algebraic rules don't have an identifier: http://sbml.org/Software/libSBML/5.18.0/docs/formatted/c-api/class_rule__t.html.  And I think in principle you can have both assignment and rate rule for the same variable?  Not sure whether that's ruled out by the no-algebraic-loops rule.

Reply:
yes, I'd even say you can have 2 same assignments for the same variable, defining equality constraint. :D

But anyway yeah this vector is correct here. :]